tools/ete/ete_tree_generator.py

```diff
@@ -0,0 +1,55 @@
+from ete3 import NCBITaxa
+import optparse
+
+
+ncbi = NCBITaxa()
```

Comment:
Should we call `ncbi.update_taxonomy_database()` as mentioned at http://etetoolkit.org/docs/latest/tutorial/tutorial_ncbitaxonomy.html#upgrading-the-local-database ?